src/views/ContactView.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <main>
+    <h1>Contact</h1>
+
+    <div id="contact-form">
+      <label for="email">Email:</label>
+      <input type="email" id="email" name="email" v-model="email" />
+      <label for="message">Message:</label>
+      <textarea id="message" name="message" v-model="message"></textarea>
+      <button @click="sendMail" id="contact-btn">SUBMIT</button>
+    </div>
```

Comment:
```suggestion
    <form id="contact-form">
      <label for="email">Email:</label>
      <input type="email" id="email" name="email" v-model="email" />
      <label for="message">Message:</label>
      <textarea id="message" name="message" v-model="message"></textarea>
      <button @click.prevent="sendMail" id="contact-btn">SUBMIT</button>
    </form>
```

Is it possible to use a form tag here instead of div? With the addition of .prevent on the click event to prevent default.

Reply:
this does not work unfortunately